paf-mvp-client-express/src/client-node.ts

```diff
@@ -46,6 +46,10 @@ export class ClientNode extends Node {
 
     this.client = new OperatorClient(operatorHost, hostName, currentPrivateKey, this.publicKeyProvider);
     this.websiteIdentityValidator = new WebsiteIdentityValidator(hostName);
+  }
+
+  async start(): Promise<void> {
```

Comment:
IMO, documenting this method is valuable here: it will help to understand the rationale of using the start instead of the constructor. 

First, I am sorry that I didn't share my perspective on how the start method should be integrated in the lifecycle of the app. It was initially designed to load resources like caches, not really for loading the routes. Here is a flow that we can refine together if you want:

1. The server is launched and set up all the routes as fast as possible including the health check one.
2. During the launch, we call the `start` to load all the necessary resources asynchronously. In this timeframe, the health check returns an HTTP 503.
3. When the start is finished, the health check returns an HTTP 200 stating that it is ready.

Reply:
If the routes aren't ready, they should return a `503` like the health check.

Reply:
Discussed offline. The subject needs more specification and to ease this PR integration we postpone this discussion.